storybook/.storybook-web/webpack.config.js

```diff
@@ -23,10 +23,5 @@ module.exports = ({ config }) => {
     ]
   });
 
-  config.resolve.alias = {
-    '@storybook/react-native': '@storybook/react',
-    'react-native': 'react-native-web'
-  };
-
-  return config;
+  return withUnimodules(config, { projectRoot: resolve(__dirname, "../../") });
```

Comment:
this, in essence- handles the entire web experience for us including potential transpilations expo thinks it will need.